scripts/jenkins/cloud/manual/Dockerfile

```diff
@@ -0,0 +1,57 @@
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE/Updates/SLE-Live-Patching/12/x86_64/update/                   live-patching
+RUN zypper addrepo --refresh http://clouddata.cloud.suse.de/repos/x86_64/SLES12-SP4-Pool/                                   pool
+RUN zypper addrepo --refresh http://clouddata.cloud.suse.de/repos/x86_64/SLES12-SP4-Updates/                                up
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/GA/standard/                                     12-SP4-GA
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/Update/standard/                                 12-SP4-Update
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP3:/GA/standard/                                     12-SP3-GA
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP3:/Update/standard/                                 12-SP3-Update
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/GA/standard/                                     12-SP2-GA
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update/standard/                                 12-SP2-Update
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP1:/GA/standard/                                     12-SP1-GA
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12-SP1:/Update/standard/                                 12-SP1-Update
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12:/GA/standard/                                         12-GA
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/                                     12-Update
+RUN zypper addrepo --refresh http://download.suse.de/ibs/SUSE:/Factory:/Head/standard/                                      factory
+
+RUN zypper --gpg-auto-import-keys refresh
+
+RUN zypper --non-interactive --gpg-auto-import-keys install \
+        autoconf \
+        automake \
+        ca-certificates-suse \
+        gcc \
+        git-core \
+        python-devel \
+        python-virtualenv \
+        python3 \
+        python3-devel \
+        python3-virtualenv \
+        sshpass \
+        tar \
+        vim \
+        vim-data \
+        wget
+
+COPY requirements.txt /tmp/requirements.txt
+
+RUN virtualenv --python python3 /opt/ansible-venv-python3
+RUN /opt/ansible-venv-python3/bin/pip install --upgrade pip
+RUN /opt/ansible-venv-python3/bin/pip install --requirement /tmp/requirements.txt
+RUN /opt/ansible-venv-python3/bin/pip install python-openstackclient python-heatclient pyyaml
+
+RUN virtualenv --python python2 /opt/ansible-venv-python2
+RUN /opt/ansible-venv-python2/bin/pip install --upgrade pip
+RUN /opt/ansible-venv-python2/bin/pip install --requirement /tmp/requirements.txt
```

Comment:
mitogen also needs to be installed by pip. However its questinable that it needs to be part of "requirements.txt" because it is only necessary for a containarized deployment.  
Probably need a new docker_requirements.txt, but for now the following should suffice: 

RUN /opt/ansible-venv-python2/bin/pip install  mitogen

and 

RUN /opt/ansible-venv-python3/bin/pip install --requirement /tmp/mitogen